processing/src/main/java/io/druid/query/groupby/epinephelinae/RowBasedGrouperHelper.java

```diff
@@ -92,7 +94,9 @@
       final int concurrencyHint,
       final LimitedTemporaryStorage temporaryStorage,
       final ObjectMapper spillMapper,
-      final AggregatorFactory[] aggregatorFactories
+      final AggregatorFactory[] aggregatorFactories,
+      @Nullable final ListeningExecutorService grouperSorter,
+      final int priority
   )
   {
     // concurrencyHint >= 1 for concurrent groupers, -1 for single-threaded
```

Comment:
1 and -1 are implemented differently; 1 is a ConcurrentGrouper with 1 slice and -1 is a non-concurrent grouper. I'm not sure if this actually makes sense but that's the way it is. It doesn't come up much in practice since it's rare to have concurrencyHint = 1.